### ISSUE TITLE
Outdated images documentation

### ISSUE THREAD

**Issue by MichalBryxi** [empress/empress-blog#123]
I think following sentence from [the documentation](https://empress-blog.netlify.app/the-editor/) is not correct anymore? Empress is not using [ember-responsive-image](https://github.com/kaliber5/ember-responsive-image), which should solved this automagically?

> Important Note: empress-blog does not currently have automatic image resizing, so it's always a good idea to make sure your images aren't gigantic files before adding them to your project.

**Comment by elwayman02** [comment]
empress-blog itself doesn't have automatic image sizing, but the Casper template provides it I believe? However, since the Casper template isn't part of this project directly, the documentation for empress-blog itself should reflect that it isn't included. That said, it would be a nice improvement to point out in the docs that the default template has this capability.

**Comment by MichalBryxi** [comment]
Agree. The reality of things is that most people will opt for default template. So even having a note in `empress-blog` README that "solution for this common thing can be found here" will be highly useful.
